### ISSUE TITLE
Reading of training set and test set

### ISSUE THREAD

**Issue by OnePl3** [radar-lab/mmfall#6]
Hello, there seems to be some problem with reading the dataset.

![image](https://github.com/radar-lab/mmfall/assets/61307360/39c89f3c-f9e5-43ca-ac56-86e527aa11b8)

**Comment by ggx159** [comment]
我也遇到了这个问题，如何解决呢

<img width="993" alt="联想截图_20240113141811" src="https://github.com/radar-lab/mmfall/assets/118532959/8f949517-c712-4c83-94f2-81d73cfe443e">
